packages/ds-healthcare-gov/src/components/Header/MenuLinks.tsx

```diff
@@ -35,7 +35,11 @@ const MenuLinks = (props: MenuLinksProps) => (
           key={link.href}
           className={`ds-u-margin--0 ${isLoginLogoutLink ? 'ds-u-border-top--1' : ''}`}
         >
-          <a href={link.href} onClick={onClick} className="hc-c-menu__link">
+          <a
+            href={link.href}
+            onClick={onClick}
+            className={`hc-c-menu__link ${link.className ?? ''}`}
```

Comment:
In other modules in this directory, we use a `classnames` utility for combining multiple optional class names, and using that here would avoid the snapshot changes that we're seeing.

Reply:
Will do.
I'd considered doing something like:
```js
[
  'hc-c-menu__link',
  link.className
].filter(Boolean).join(' ');
```
but that felt heavy handed for just two potential classes -- I generally only use that when I'm stacking quite a few.